packages/protocol/src/utils/index.ts

```diff
@@ -16,7 +16,5 @@
 export * from './array-util';
 export * from './disposable';
 export * from './event';
-// we do not export test-util to avoid a dependency on test frameworks such as chai
-// however, adopters can still access the file by accessing it through the complete path
-// export * from './test-util';
+export * from './test-util';
```

Comment:
We should create a `.indexignore` file in the utils directory with the following content to avoid the unwanted export:
```
# we do not export test-util to avoid a dependency on test frameworks such as chai
# however, adopters can still use the functionality by importing it through the complete path
test-util.ts
```